Content.Server/Lightning/LightningTargetSystem.cs

```diff
@@ -22,17 +22,17 @@ public override void Initialize()
 
     private void OnHittedByLightning(EntityUid uid, LightningTargetComponent component, ref HittedByLightningEvent args)
     {
-        Log.Debug("��, ���� ������� ������");
-        if (!component.LightningExplode)
-            return;
-        Log.Debug("�������� ����������");
-        _explosionSystem.QueueExplosion(
-            uid,
-            component.ExplosionPrototype,
-            component.TotalIntensity,
-            component.Dropoff,
-            component.MaxTileIntensity
-            );
-        Log.Debug("���������");
+        //Log.Debug("��, ���� ������� ������");
+        //if (!component.LightningExplode)
+        //    return;
+        //Log.Debug("�������� ����������");
+        //_explosionSystem.QueueExplosion(
```

Comment:
There are currently 2 bugs that I haven't found how to fix yet. They are... specific

The first occurs when a Lightning Strike triggers the HittingByLightningEvent event and the LightningTarget component attempts to make an explosion. In dev arena everything works stable, but on mapinit maps it gives the following server error:
![image](https://github.com/space-wizards/space-station-14/assets/96445749/72aa0d23-07a5-47d3-a851-92e93e1230ab)

Reply:
Error Stack Trace
```
Robust.Shared.Utility.DebugAssertException
  HResult=0x80131500
  Message = Root node count is 2 instead of 1.
  Source  = Robust.Shared
  Stack trace:
   in Robust.Shared.Utility.DebugTools.Assert(Boolean condition, AssertInterpolatedStringHandler& message) in C:\Users\edwar\Desktop\ss14 SpriteWork\Local SS14 Client Full\ss off eng\ed-station-14\RobustToolbox\Robust.Shared\Utility\DebugTools.cs:stroke 141
   in Robust.Server.GameStates.ServerGameStateManager.<>c__DisplayClass39_0.<GetPVSData>b__0(Int32 i) in C:\Users\edwar\Desktop\ss14 SpriteWork\Local SS14 Client Full\ss off eng\ed-station-14\RobustToolbox\Robust.Server\GameStates\ServerGameStateManager.cs:stroke 298
   in System.Threading.Tasks.Parallel.<>c__DisplayClass19_0`1.<ForWorker>b__1(RangeWorker& currentWorker, Int32 timeout, Boolean& replicationDelegateYieldedBeforeCompletion)
```

Reply:
there was an assumption that the explosions have a chance of breaking the local server.  But on full-fledged servers, it seems, there will be no such error.  I'd like to hope so, but it's worth double-checking.

Reply:
@deltanedas 
maybe you know something about this problem?

Reply:
bug reproduction: 
run any playable map (bagel) via mapping + mapinit.  Spawn "Tesla", watch the destruction of the station and the inevitable crash of the server.

Reply:
Ok, as tests have shown, there are no errors if tesla is launched in the release version of the build. Conclusion - this is not a problem of tesla, but of debug mode, where something often breaks.